### PR TITLE
Master

### PR DESCRIPTION
**Pull request by SamDecrock**
Added rinfo to routing callbacks and added option to set TTL when responding to DNS requests

### CODE REVIEW

**Comment by roccomuso** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dnsjack",
-  "version": "0.3.0",
+  "version": "0.4.0",
```
Comment (mafintosh) [inline package.json]: could you remove the version bump?

File: index.js [hunk 2]
```diff
@@ -228,14 +233,25 @@ exports.createServer = function(proxy) {
 
 		if (!route) return onproxy();
 
-		route(domain, function(err, to) {
+		route(routeData, function (err, to) {
+			var toIp,
+				ttl;
```
Comment (mafintosh) [inline index.js]: some indentation thing looks off here. in general i prefer multiple var statements but if you fix the indentation thats fine too

Comment (SamDecrock) [inline index.js]: done!

File: package.json (hunk 1 above)
Comment (SamDecrock) [inline package.json]: done!

**Comment by roccomuso** [comment]
So ready to merge?

**Comment by mafintosh** [comment]
Looks good yea! Out in 1.0.0

**Pull request merged**